admission-controller/values.schema.json

```diff
@@ -0,0 +1,1004 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "http://example.com/example.json",
```

Comment:
Hi @michael-zhou-lw, the `$id` field here is supposed to identify this schema uniquely. Perhaps something like `https://lacework.github.io/helm-charts/admission-controller/values.schema.json`?

Reply:
@cirego I've created another PR to fix this and update the README, could you have a look please? https://github.com/lacework/helm-charts/pull/9